### DISCUSSION TITLE
How to customize the colors of file names and directory names?

### DISCUSSION THREAD

**Discussion by hopezh** [lsd-rs/lsd#996] [Q&A] [answered]
As shown below, may I ask how to customize the colors of the names and icons for files and that for directories for LSD? Thanks.

![image](https://github.com/lsd-rs/lsd/assets/23435810/15960719-e1e8-45ac-86f5-23660841efde)

**Comment by zwpaper** [comment, marked as answer]
we currently using LSCOLORS for the filename colors, check https://geoff.greer.fm/lscolors/

**Reply by hopezh** [reply]
Thanks. May I ask how to do the same for lsd installed in windows, considering that lscolors is for linux?

**Reply by PGScully** [reply]
Use the link provided to create your colour scheme, then create an environment variable LS_COLORS using the contents of the 'Linux LS_COLORS' field.
